src/sentry/static/sentry/app/stores/globalSelectionStore.jsx

```diff
@@ -139,10 +140,12 @@ const GlobalSelectionStore = Reflux.createStore({
   },
 
   updateLocalStorage() {
+    // Do nothing if no org is loaded or user is not an org member
```

Comment:
Can we clarify how/why we know that the user is not an org member?

Reply:
Oh, is the assumption on the line below not clear/correct? i.e. not in their organizations store?

Reply:
The assumption is correct, I think it'd be good to clarify the reason why organization wouldn't exist in OrganizationStore. e.g. If user is a superuser, `OrganizationsStore` will only contain orgs that they are a member of, while the current organization could be one they can access, but not be a member of.

Reply:
Ok